src/picongpu/include/plugins/common/txtFileHandling.hpp

```diff
@@ -46,28 +47,46 @@ using namespace boost::filesystem;
     bool restoreTxtFile( std::ofstream& outFile, std::string filename,
                          uint32_t restartStep, const std::string restartDirectory )
     {
-        if( outFile.is_open() )
-            outFile.close();
-
+        /* get restart time step as string */
         std::stringstream sStep;
         sStep << restartStep;
 
+        /* set location of restart file and output file */
         path src( restartDirectory + std::string("/") + filename +
                   std::string(".") + sStep.str() );
         path dst( filename );
 
-        copy_file( src,
-                   dst,
-                   copy_option::overwrite_if_exists );
-
-        outFile.open( filename.c_str(), std::ofstream::out | std::ostream::app );
-        if( !outFile )
+        /* check whether restart file exists */
+        std::ifstream restartFile(src.c_str());
+        if(! restartFile.good())
         {
-            std::cerr << "[Plugin] Can't open file '" << filename
-                      << "', output disabled" << std::endl;
-            return false;
+            /* restart file does not exists */
+            restartFile.close();
+            std::cerr << "Plugin restart file: \n \t" << src << "\n was not found." << std::endl;
+            std::cerr << "Starting plugin from current time step." << std::endl;
+            return true;
+        }
+        else
+        {
+            /* restart file found - fix output file created at restart */
+            restartFile.close();
+
+            if( outFile.is_open() )
+                outFile.close();
+
+            copy_file( src,
+                       dst,
+                       copy_option::overwrite_if_exists );
+
+            outFile.open( filename.c_str(), std::ofstream::out | std::ostream::app );
+            if( !outFile )
+            {
+                std::cerr << "[Plugin] Can't open file '" << filename
```

Comment:
this line on the other hand is an *unexpected* state. even if log `INPUT_OUTPUT` would apply somehow, I think `cerr` is appropriate.